City-of-Helsinki/drupal-helfi-asuminen
# UHF-4334: Watson chatbot

How to test:

1. set up Asuminen instance
2. checkout this branch: `git checkout UHF-4334_watson_chatbot`
3. get the equivalent branch of the Platform Config module: `composer require drupal/helfi_platform_config:dev-UHF-4334_watson_chatbot` (see PR: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/255)
4. import config: `make drush-cim`
5. create a new content page to one of the URLs defined in the block config file (for example: `/vuokra-asunnot`)
6. navigate to the page and make sure that the chatbot shows up in the bottom right corner
